sample-build-scripts/react-native/fix-cocoapods-and-node-build/appcenter-post-clone.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# This post clone script prepares the build process
+# in App Center for React Native 0.60 projects by
+# resolving a few build errors.
+#
+# Put this script in your ios/ folder and commit.
+#
+# Please mind the version numbers so that they
+# correspond to your Podfile and package.json.
+
+echo "Uninstalling all cocoapods versions"
```

Comment:
Why do we need to install cocoapods 1.7.5 in post-clone script?
Build machines contain this version of cocoapods by default.

Reply:
Context: https://github.com/microsoft/appcenter/issues/214#issuecomment-492124627

The error I got before adding this to the build script, was:

```
##[section]Starting: Pod install
==============================================================================
Task         : CocoaPods
Description  : Install CocoaPods dependencies for Swift and Objective-C Cocoa projects
Version      : 0.151.1
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/package/cocoapods
==============================================================================
[command]/usr/local/lib/ruby/gems/2.6.0/bin/pod --version
1.7.5
[command]/usr/local/lib/ruby/gems/2.6.0/bin/pod install --repo-update

[!] Invalid `Podfile` file: cannot load such file -- /Users/vsts/agent/2.155.1/work/1/s/node_modules/@react-native-community/cli-platform-ios/native_modules.

 #  from /Users/vsts/agent/2.155.1/work/1/s/ios/Podfile:2
 #  -------------------------------------------
 #  platform :ios, '9.0'
 >  require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 #  
 #  -------------------------------------------
##[error]The process '/usr/local/lib/ruby/gems/2.6.0/bin/pod' failed with exit code 1
##[error]The 'pod' command failed with error: The process '/usr/local/lib/ruby/gems/2.6.0/bin/pod' failed with exit code 1
##[section]Finishing: Pod install
```